doc/user/pim.rst

```diff
@@ -389,10 +389,15 @@ cause great confusion.
 
    Display IGMP interface information.
 
-.. index:: show ip igmp join
-.. clicmd:: show ip igmp join
+.. index:: show ip igmp [vrf NAME] join [json]
+.. clicmd:: show ip igmp [vrf NAME] join [json]
 
-   Display IGMP static join information.
+   Display IGMP static join information for a specific vrf.
```

Comment:
I think it would just be sufficient to state in this description something like `if the word all is used for the vrf name then it displays all vrf's instead of adding a new ..index...